### PR TITLE
feat(datepicker): add prev/next year buttons

### PR DESCRIPTION
**Pull request by shijiatongxue**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Feature 

### PR description

- DatePicker add prev/next year buttons #260

### Changelog
🇨🇳 Chinese
- DatePicker 新增年份切换按钮 #260
- DatePicker [优化范围选择年月切换逻辑](https://bytedance.feishu.cn/docx/doxcnckfGLOJVV0HDUIcsca7QHf ) #260

---

🇺🇸 English
- DatePicker add prev/next year buttons #260
- DatePicker [optimized year and month switching logic of range type picker](https://bytedance.feishu.cn/docx/doxcnckfGLOJVV0HDUIcsca7QHf ) #260

### Checklist
- [x] Test or no need
- [x] Changelog or no need

### Demo

```jsx
import React from 'react';
import { DatePicker } from '@douyinfe/semi-ui';

export default function App() {
    return (
        <div>
            <h4>type=date</h4>
            <DatePicker />
            <h4>type=dateRange</h4>
            <DatePicker type="dateRange" defaultPickerValue="2021-12" />
            <h4>type=dateTimeRange</h4>
            <DatePicker type="dateTimeRange" />
            <h4>type=dateRange + compact</h4>
            <DatePicker type="dateRange" density="compact" />
        </div>
    );
}
```

### CODE REVIEW

File: packages/semi-ui/datePicker/navigation.tsx [hunk 1]
```diff
@@ -77,43 +79,63 @@ export default class Navigation extends PureComponent<NavigationProps> {
         const iconBtnSize = density === 'compact' ? 'default' : 'large';
         const btnNoHorizontalPadding = true;
         const buttonSize = density === 'compact' ? 'small' : 'default';
+        const leftPanel = panelType === strings.PANEL_TYPE_LEFT;
```
Comment (pointhalo) [inline packages/semi-ui/datePicker/navigation.tsx:82]: isLeftPanel 更符合语义一些？

File: packages/semi-icons/src/icons/IconDoubleChevronRight.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { convertIcon } from '../components/Icon';
+
+function SvgComponent(props: React.SVGProps<SVGSVGElement>) {
+    return (
+        <svg
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            width="1em"
+            height="1em"
```
Comment (pointhalo) [inline packages/semi-icons/src/icons/IconDoubleChevronRight.tsx:11]: 这个写死的width、height 1em是符合预期的吗，其他svg转出来的comonent也是一样的吗

File: packages/semi-ui/datePicker/navigation.tsx (hunk 1 above)
Comment (shijiatongxue) [reply packages/semi-ui/datePicker/navigation.tsx:82]: 有种说法是布尔值不用加 is

File: packages/semi-foundation/datePicker/datePicker.scss [hunk 3]
```diff
@@ -94,6 +94,15 @@ $module: #{$prefix}-datepicker;
                 min-height: $height-datepicker_timepicker_header_min;
             }
         }
+
+        // 为了防止 scrollList 因为 weeks 变化高度发生变化导致年月可能发生滚动
+        // In order to prevent scrollList from scrolling due to changes in the height of weeks, the year and month may be scrolled
+        &[x-yam-open-type="left"],
```
Comment (pointhalo) [inline packages/semi-foundation/datePicker/datePicker.scss:100]: yam这个不要简写？

**Pull request merged**